Engine/Utilities.cs

```diff
@@ -1645,4 +1646,47 @@ object parseElement(object parent, XElement parentElement, XElement element)
             return scene as Scene;
         }
     }
+
+    public static class ChromeProfilerReader
```

Comment:
1) move methods inside ProfilerCore
2) explore other browsers options (for example firefox or opera https://www.reddit.com/r/firefox/comments/f6gcpk/converting_chrome_tracer_results_to_firefox/) and expose methods like DumpChrome, DumpOpera, DumpFirefox etc